### DISCUSSION TITLE
How to reduce outliers after having performed hierarchical topic modeling?

### DISCUSSION THREAD

**Discussion by HastyBeaver** [MaartenGr/BERTopic#1728] [Q&A] [answered]
Hi everybody,

After finding topics and probabilities using the following code:
`topics, probs = topic_model.fit_transform(train_documents, embeddings)`
I wanted to perform hierarchical topic reduction. After visualising the hierarchy, I decided to merge a few topics together using the `merge_topics` function. I also visualized the pd.Dataframe printed by `get_topic_info`, and, obviously, I can see the new topics as well as the new outliers. 
Now, I want to apply `reduce_outliers`, but it seems that this function requires the `topics` list as a parameter, which is supposedly the topic list _after_ the hierarchical reduction.
How can I obtain this? I think I may be missing something, but I can't seem to find a solution.

Thank you!

**Comment by MaartenGr** [comment, marked as answer]
You can access any updated topics with `topioc_model.topics_` and use those for further calculations/processes.
